routing/routes.go

```diff
@@ -20,7 +20,7 @@ type Routes struct {
 	Routes []Route
 }
 
-// A GenericError is the default error message that is generated.
+// GenericError: any other kind of error
```

Comment:
@xeviknal @aljesusg I added the error "BadRequest" in swagger because it's quite common, and also changed the descriptions of the errors to something shorter, more direct (tell me if you don't like I can revert it).